src/Symfony/Bridge/Twig/Tests/Command/DebugCommandTest.php

```diff
@@ -293,6 +294,30 @@ public function testWithFilter()
         $this->assertNotSame($display1, $display2);
     }
 
+    /**
+     * @dataProvider provideCompletionSuggestions
+     */
+    public function testComplete(array $input, array $expectedSuggestions)
+    {
+        $projectDir = \dirname(__DIR__).\DIRECTORY_SEPARATOR.'Fixtures';
```

Comment:
We should skip this test if autocomplete is not available (see other similar tests in the code base).

Reply:
Found and added 
```
        if (!class_exists(CommandCompletionTester::class)) {
            $this->markTestSkipped('Test command completion requires symfony/console 5.4+.');
        }

```